easybuild/easyconfigs/r/R-bundle/Ignore-BI_COMM_GLOBAL-on-intel.patch

```diff
@@ -0,0 +1,43 @@
+--- pbdBASE.orig/src/export_blacs/pkg_tools.c   2015-06-02 15:25:57.705288000 +0200
++++ pbdBASE/src/export_blacs/pkg_tools.c        2015-06-02 15:25:29.470539000 +0200
+@@ -25,7 +25,9 @@
+        BLACS_APTS.BI_AuxBuff = &BI_AuxBuff;
+        BLACS_APTS.BI_MyContxts = BI_MyContxts;
+        BLACS_APTS.BI_SysContxts = BI_SysContxts;
++#ifndef __INTEL_COMPILER
```

Comment:
Unsure. I ran the following on the easyconfig dir and didn't see a better one currently in use:

`find . -name '*.patch' | xargs grep INTEL`